cypher/cypher-docs/src/docs/dev/ql/administration/security/role-management-syntax.asciidoc

```diff
@@ -55,6 +55,13 @@ CREATE OR REPLACE ROLE name [AS COPY OF name]
 | <<administration-security-administration-dbms-privileges-role-management, CREATE ROLE>> and
 <<administration-security-administration-dbms-privileges-role-management, DROP ROLE>>
 
+| [source, cypher, role=noplay]
+----
+RENAME ROLE name [IF EXISTS] TO otherName
```

Comment:
`create role ... as copy of ...` has `name` in both places, maybe that too should have `otherName` or this have `name` to be consistent?

Reply:
I think otherName is clearer. If you are putting the same name in both places the commands will fail after all. I will update the `as copy of`